src/modules/75-cf/components/AuditLogs/EventSummary.module.scss

```
@@ -36,3 +36,9 @@
 .backdrop {
   background-color: rgba(16, 22, 26, 0.4) !important;
 }
+
+.yamlDiffBtn {
+  color: var(--grey-800) !important;
+  font-size: 10px !important;
```

```suggestion
  font-size: var(--font-size-xsmall) !important;
```
this is a nitpick and not at all a blocker